notebooks/py_scripts/10_Spike_SortingV1.py

```diff
@@ -94,13 +101,20 @@
 
 sgs.ArtifactDetection()
 
-# insert SpikeSortingSelection. again use `insert_selection` method
+# insert SpikeSortingSelection. again use `insert_selection` method.
+#
+# We tested mountainsort4, mountainsort5, kilosort2_5, kilosort3, and ironclust.
+# when using mountainsort5, pip install 'mountainsort5'
```

Comment:
In my conda environment, I was not able to `pip install mountainsort4` directly because of its dependency on `pybind11`.  I made a cell like 

```
#!pip install pybind11 
!pip install mountainsort4
```